test/BuildTools.Tasks.Tests/UnzipArchiveTest.cs

```diff
@@ -149,6 +149,30 @@ public void DoesNotOverwrite()
             Assert.Equal(contents, File.ReadAllText(path));
         }
 
+        [Fact]
+        public void ItNormalizesBacklashesInPath()
```

Comment:
Did we test this against Ubuntu or some other non-windows OS?

Reply:
MacOS. I confirmed this by running on tests before/after on my macbook